std/file.d

```diff
@@ -2913,6 +2913,9 @@ public:
     void popFront(){ impl.popFront(); }
 
 }
+
+alias FollowSymlink = Flag!"FollowSymlink";
+
```

Comment:
The idiom is, `alias FollowSymlink = Flag!"followSymlink"`, so that you get "correct" casing for all uses, i.e. `FollowSymlink.no` and `No.followSymlink`.